setup.py

```diff
@@ -29,6 +29,7 @@
 from setuptools import setup
 import imp
 import sys
+import versioneer
 
 version = imp.load_source('trollsched.version', 'trollsched/version.py')
```

Comment:
This line can be removed now, right?

Reply:
Ahh, yeah, right

Reply:
Make sure you remove the `import imp` line too.

Reply:
Oh yes, thanks. Done